libseccomp/src/lib.rs

```diff
@@ -820,6 +820,11 @@ impl ScmpSyscall {
         Ok(name)
     }
 }
+impl From<i32> for ScmpSyscall {
```

Comment:
Could you add the doc comment as follows?
In addition, I think it would be better to rename `nr` to `num` for easy understanding. I'm not sure which one is best. But, if you are ok, could you fix all the `nr` words to `num`?

```
/// Creates a `ScmpSyscall` from the specified syscall number.
///
/// # Arguments
///
/// * `num` - The number of syscall
```

Reply:
Somehow I prefer `nr` for `ScmpSyscall { }` because it is used in `__(S|P|)NR_<scall>`, `__NR_SCMP_UNDEF`, ….

Reply:
I understand :). I agree with using `nr`.  Could you add only doc comments?